training/training_code/training_requirements.txt

```diff
@@ -0,0 +1 @@
+tensorflow == 1.12.2
```

Comment:
If running in a WML environment, we actually don't need to pip install the framework as it's already available.